php/common.php

```diff
@@ -174,7 +176,8 @@ function get_settings($filename = "")
     // отчёты
     $config['reports'] = [
         'auto_clear_messages' => $ini->read('reports', 'auto_clear_messages', 0),
-        'exclude_forums_ids' => $ini->read('reports', 'exclude', ''),
+        'exclude_forums_ids' => $ini->read('reports', 'exclude_forum', ''),
+        'exclude_clients_ids' => $ini->read('reports', 'exclude_client', ''),
```

Comment:
A bit confusing naming. Singular form hints that the value is rather a boolean than an array. Maybe keep it in the INI with the same name as in the object? I.e., `exclude_forums_ids`, `exclude_clients_ids`. Not insisting though.

Reply:
fixed